viadot/flows/genesys_to_adls.py

```diff
@@ -137,6 +139,8 @@ def __init__(
             report_url (str, optional): The url of report generated in json response. Defaults to None.
             report_columns (List[str], optional): List of exisiting column in report. Defaults to None.
             conversationId_list (List[str], optional): List of conversationId passed as attribute of GET method. Defaults to None.
+            mapping_dict (dict, optional): Mapping dictionary from user in json format. Defaults to None.
```

Comment:
Please add maybe 1 example of how this `mapping_dict` should look. It can be added here in the docstring for `mapping_dict`. Could you add in every place where this description appears?